### ISSUE TITLE
fpv.wtf is not working

### ISSUE THREAD

**Issue by michi20blau** [fpv-wtf/wtfos-configurator#206]
hello, i have a problem with fpv.wtf the glasses v2 worked great! but with the air unit the root worked but when i try to connect it always throws me off the internet site what can i do? lg.

**Comment by stylesuxx** [comment]
What does "throws me off the internet site" mean? Does your browser crash? Or what is happening exactly?

**Comment by michi20blau** [comment]
I did everything like on the video, so far everything worked! but get no osd what am I doing wrong?

**Comment by stylesuxx** [comment]
Well, probably you are missing something. I suggest you hop on Discord for debugging session. Otherwise post more infos: BF version, CLI dump... Which version of msp-osd are you running on the googles/air unit? Are you seeing the "waiting for OSD" text?

**Comment by stylesuxx** [comment]
Then upgrade on air unit side - they should always be upgraded in parallel. Also - does the custom DJI OSD work at all (or did it work before adjusting settings)?
Please post CLI dump, also post screenshot of your port tab from BF.

**Comment by michi20blau** [comment]
yes before the root it worked just what dji supports! I also tried to delete msp again but now nothing is displayed for me with osd not even the original ones from dji? txt I added!

[BTFL_cli_ProTek35_20221017_185441.txt](https://github.com/fpv-wtf/wtfos-configurator/files/9811442/BTFL_cli_ProTek35_20221017_185441.txt)

![Betaflight Configurator 18 10 2022 16_30_18](https://user-images.githubusercontent.com/41871815/196460075-7b41ebf9-05e5-46df-bc5e-56624eefa924.png)

**Comment by michi20blau** [comment]
yes I'm quite sure the osd is active in the glasses! I'm also sure that the original dji osd worked before the root! what does jump troubleshooting session on discord mean?

**Comment by stylesuxx** [comment]
@michi20blau any update on this issue?

**Comment by michi20blau** [comment]
Hello, finally installed msp-osd successfully! When I run the FakeHD, does the Betaflight menu (Settings) appear normally? Or is that all postponed? And how is the post-flight record flight time, etc. is all this displayed normally?

**Comment by stylesuxx** [comment]
Great to hear - what was the issue in the end? Menu and post flight stats should all appear normally.

**Comment by michi20blau** [comment]
I have no idea what exactly the problem was. I deleted everything 3 times and reinstalled and installed root etc. and it's been working ever since! no idea what went wrong the first two times?

**Comment by stylesuxx** [comment]
Hmm, interesting. Hard to say after the fact. We should have checked if the actual MSP service was running at all...
Happy to hear it works, closing this issue.
